Python/ceval.c

```diff
@@ -2599,6 +2599,11 @@ _PyEval_GetBuiltinId(_Py_Identifier *name)
 PyObject *
 PyEval_GetLocals(void)
 {
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
+        "PyEval_GetLocals() is deprecated in Python 3.14 "
+        "and will be removed in 3.16.", 1)) {
```

Comment:
This is the convention for us:
```suggestion
    if (PyErr_WarnEx(PyExc_DeprecationWarning,
        "PyEval_GetLocals() is deprecated in Python 3.14 "
        "and will be removed in 3.16.", 1) < 0) {
```